lib/test/cdo/contact_rollups/test_pardot_helpers.rb

```diff
@@ -33,4 +33,22 @@ def test_raise_if_response_error_no_error
 
     assert_nil PardotHelpersTest.send(:raise_if_response_error, pardot_ok)
   end
+
+  def test_try_with_exponential_backoff_one_retry
+    tries = 0
+    max_tries = 2
+    assert_raises RuntimeError do
+      PardotHelpersTest.try_with_exponential_backoff(max_tries, [RuntimeError]) do
```

Comment:
Sorry this is more of a question hour than a proper PR review :) Why do we use the test class here instead of just using the class itself like we do elsewhere?

Reply:
No problem at all 😄 Since `PardotHelpers` is a module, we have to test it by mixing it into a test class. In this case, the module is mixed into `PardotHelpersTest` using `extend`. ([extend vs. include](https://stackoverflow.com/questions/156362/what-is-the-difference-between-include-and-extend-in-ruby))